libgalois/src/analytics/pagerank/pagerank-pull.cpp

```diff
@@ -191,24 +199,27 @@ ComputePRTopological(Graph* graph, katana::analytics::PagerankPlan plan) {
     katana::do_all(
         katana::iterate(*graph),
         [&](const GNode& src) {
-          auto& sdata = graph->GetData<PagerankValueAndOutDegree>(src);
+          //auto& sdata = graph->GetData<PagerankValueAndOutDegree>(src);
           float sum = 0.0;
 
           for (auto jj : graph->edges(src)) {
             auto dest = graph->GetEdgeDest(jj);
-            auto& ddata = graph->GetData<PagerankValueAndOutDegree>(dest);
-            sum += ddata.value / ddata.out;
+            //auto& ddata = graph->GetData<PagerankValueAndOutDegree>(dest);
+            //sum += ddata.value / ddata.out;
+            sum += (*node_data)[*dest].value / (*node_data)[*dest].out;
```

Comment:
Try this instead:
```cpp
auto& ddata = (*node_data)[*dest];
sum += ddata.value / ddata.out;
```

Reply:
tried.

Reply:
I didn't mean for performance. I meant the code reads better that way.

Reply:
yes. Oops I thought that I modified this ..